pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -821,24 +821,22 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes).To(BeEmpty())
 		})
 		It("should reject bigger guest memory than the memory limit", func() {
-			vmi := api.NewMinimalVMI("testvmi")
-			guestMemory := resource.MustParse("128Mi")
-
+			vmi := newBaseVmi()
 			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{
 				k8sv1.ResourceMemory: resource.MustParse("64Mi"),
 			}
-			vmi.Spec.Domain.Memory = &v1.Memory{Guest: &guestMemory}
 
 			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec, config)
 			Expect(causes).To(HaveLen(1))
-			Expect(causes[0].Field).To(Equal("fake.domain.memory.guest"))
+			Expect(causes[0].Field).To(Equal("fake.domain.resources.requests.memory"))
 		})
 		It("should allow bigger guest memory than the memory limit if vmRolloutStrategy is set to LiveUpdate", func() {
 			kvConfig := kv.DeepCopy()
 			kvConfig.Spec.Configuration.VMRolloutStrategy = ptr.To(v1.VMRolloutStrategyLiveUpdate)
 			testutils.UpdateFakeKubeVirtClusterConfig(kvStore, kvConfig)
 
-			vmi := api.NewMinimalVMI("testvmi")
+			vmi := newBaseVmi()
+			vmi.Spec.Domain.Resources.Requests = nil
```

Comment:
Could you please elaborate why is this assignment required?

Reply:
the `newBaseVmi` helper assign 512Mi request memory, which causes the test to fail unless we reinitialize to nil.

```
<[]v1.StatusCause | len:1, cap:1>: [
          {
              Type: "FieldValueInvalid",
              Message: "fake.domain.resources.requests.memory '512Mi' is greater than fake.domain.resources.limits.memory '64Mi'",
              Field: "fake.domain.resources.requests.memory",
          },
      ]
```

Reply:
Then you can use `libvmi.New` instead of `newBaseVmi`.